dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry10/internal/rest/TelemetryContainerFilter.java

```diff
@@ -208,6 +195,36 @@ public void filter(final ContainerRequestContext request, final ContainerRespons
         }
     }
 
+    private static String getRoute(final ContainerRequestContext request, Class<?> resourceClass, Method resourceMethod) {
+
+        String route = ROUTE_CACHE.getRoute(resourceClass, resourceMethod);
+
+        if (route == null) {
+
+            int checkResourceSize = request.getUriInfo().getMatchedResources().size();
+
+            // Check the resource size using getMatchedResource()
+            // A resource size > 1 indicates that there is a subresource
+            if (checkResourceSize == 1) {
```

Comment:
```suggestion
            int resourceCount = request.getUriInfo().getMatchedResources().size();

            // Check the resource count using getMatchedResource()
            // A resource count > 1 indicates that there is a subresource
            // We can't currently compute the route correctly when subresources are used
            if (resourceCount == 1) {
```

Please make this change in the 1.1 version of `TelemetryContainerFilter` as well